src/state/experiments/selectors.js

```diff
@@ -1,2 +1,29 @@
 export const getExperimentsCount = state =>
   state.get("experiments").get("items").size;
+
+export const getFilteredExperimentsByDate = (state) => {
+  let filteredExperiments = [];
+  const experiments = state.get("experiments").get("items"); 
+  const selectedStartTimestamp = getStartDatepickerTimestamp(state);
+  const selectedEndTimestamp = getEndDatepickerTimestamp(state);
+
+  for (let i = 0; i < experiments.size; i++) {
+    const experimentStartDate = experiments.getIn([i, "start_date"])
+    const experimentEndDate = experiments.getIn([i, "end_date"]);
+
+    if (experimentStartDate >= selectedStartTimestamp && selectedEndTimestamp >= experimentEndDate) {
+      filteredExperiments.push(experiments.getIn([i, "name"]))
```

Comment:
Wait if you only push in the name, how will you access all the other fields you want in the component?